gallery/src/pageConfig.ts

```diff
@@ -73,34 +73,34 @@ const pageConfig: PageConfig = {
     'Form Validation Guidelines': FormValidationPage
   },
   'React Components': {
+    NxAlert: NxAlertComponentsPage,
+    NxBackButton: NxBackButtonPage,
     NxButton: NxButtonPage,
     NxCheckbox: NxCheckboxPage,
-    NxRadio: NxRadioPage,
-    NxTextInput: NxTextInputPage,
-    NxBackButton: NxBackButtonPage,
-    NxLoadingSpinner: NxLoadingSpinnerPage,
+    NxDropdown: NxDropdownPage,
+    NxFilterInput: NxFilterInputPage,
+    NxFontAwesomeIcon: NxFontAwesomeIconPage,
     NxLoadError: NxLoadErrorPage,
+    NxLoadingSpinner: NxLoadingSpinnerPage,
```

Comment:
I think this should be one line down?

Reply:
> I think this should be one line down?

I fixed as part of: https://github.com/sonatype/sonatype-react-shared-components/pull/19/commits/49adb42cab53cc92a0a462713c993db778241c47